docs/sofe-api.md

```diff
@@ -117,3 +118,93 @@ System.config({
   }
 }
 ```
+
+### API
+#### getAllManifests(): Promise
+Get manifest information for available services.
+```javascript
+import { getAllManifests } from 'sofe';
+
+getAllManifests().then(manifests => {
+	manifests === {
+		flat: {
+			// Flat map of available services and urls
+		},
+		all: {
+			// Tree structure of all manifests
+		}
+	}
+});
+```
+
+#### isOverride(): Boolean
+Returns whether or not any services have been overriden by local or
```

Comment:
Would it be useful for this api also to optionally accept a serviceName param so that you could check for a specific service?